install/update_2.6_2.7.php

```diff
@@ -89,4 +89,12 @@ function plugin_formcreator_update_2_7(Migration $migration) {
       $DB->query("UPDATE `glpi_plugin_formcreator_questions` SET `name`='$name' WHERE `id` = '$id'");
    }
 
+   // Upgrade tables to innodb
+   $result = $DB->listTables('glpi_plugin_formcreator_%', ['engine' => 'MyIsam']);
+   if ($result) {
+      while ($table = $result->next()) {
+         $migration->displayMessage("Migrating {$table['TABLE_NAME']} to InnoDB...");
+         $DB->query("ALTER TABLE {$table['TABLE_NAME']} ENGINE = InnoDB");
```

Comment:
I'm not sure about the migration process, the core scripts (https://github.com/glpi-project/glpi/blob/9.3/bugfixes/scripts/innodb_migration.php) already covers plugins.

Reply:
Hi

indeed. I'll remove the migration step.